### DISCUSSION TITLE
Adding few params to Signin method

### DISCUSSION THREAD

**Discussion by samymih** [nextauthjs/next-auth#5203] [Ideas]
### Description 📓

Hey, i'm making a discord bot panel and i want to open a new window (popup) with the signIn function with parameters like :
```JS
signIn('discord', {
  popup: {
    width: 600,
    height: 600
  }
})
```
it can be very good if someone want to open a popup for the login.
I don't think there is potential problems to do this.
I can contribute to it and help for the code :)

I hope my idea will be taken 😄

### How to reproduce ☕️

```JS
window.open("url", "popup", "width=600,height=600")
```

### Contributing 🙌🏽

Yes, I am willing to help implement this feature in a PR

**Comment by LuciferHasBeenSpotted** [comment]
Hope they add, it will be really usefull

**Comment by HidaSharker** [comment]
The idea is pretty good and "modern" and I don't think it's hard to implement it
